### ISSUE TITLE
Need c# client

### ISSUE THREAD

**Issue by wangronghua** [seata/seata#1296]
Need c# client

**Comment by slievrly** [comment]
@wangronghua Are you willing to participate in the net core development of seata?

**Comment by wangronghua** [comment]
yes i would发自我的华为手机-------- 原始邮件 --------主题：Re: [seata/seata] Need c# client (#1296)发件人：jimin 收件人：seata/seata 抄送：wangronghua ,Mention @wangronghua Are you willing to participate in the net core development of seata?

—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or mute the thread.
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/seata/seata/issues/1296?email_source=notifications\u0026email_token=AEYFS7GVE3N2LYSXL4VGVVDP7HBJDA5CNFSM4ICZKWQKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGODZ3QREA#issuecomment-511117456",
"url": "https://github.com/seata/seata/issues/1296?email_source=notifications\u0026email_token=AEYFS7GVE3N2LYSXL4VGVVDP7HBJDA5CNFSM4ICZKWQKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGODZ3QREA#issuecomment-511117456",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by wangronghua** [comment]
What am I going to do?

**Comment by LMing123** [comment]
@slievrly  Is there a plan to develop a C # client？

**Comment by amingo123** [comment]
@slievrly please invite me to the C# seata team.
